admin/src/scenes/volontaires-responsible/index.js

```diff
@@ -1,10 +1,10 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Switch, Route } from "react-router-dom";
 
 import List from "./list";
 import View from "./view";
 
-export default () => {
+export default ({ ...props }) => {
```

Comment:
We don't use it, no need to change

```suggestion
export default () => {
```